src/components/DataViewer.vue

```diff
@@ -13,6 +13,7 @@
           data-cy="weaverbird-data-viewer-table"
           v-resizable="{
             columns: columnNames,
+            labelClass: 'data-viewer__header-label',
             classes: {
               table: 'data-viewer-table--resizable',
               handler: 'data-viewer__header-cell--resizable',
```

Comment:
Why this class is not part of the `classes` prop below ?
Also I think that "label" is a little to vague, may be we can find a slightly more precise name like "headerLabels" ?

Reply:
I will update the property name, it should be labelTargetClass because what's under classes are the classes added to element while the labelClass, is the DOM element to target ;)